src/main/scala/de/upb/cs/swt/delphi/crawler/storage/ElasticReachablePreflightCheck.scala

```diff
@@ -44,8 +45,14 @@ object ElasticReachablePreflightCheck extends PreflightCheck {
 
     val f = (client.execute {
       nodeInfo()
-    } map { i => Success(configuration)
-    } recover { case e => Failure(e)
+    } map { i => {
+      if(configuration.usingInstanceRegistry) InstanceRegistry.sendMatchingResult(isElasticSearchReachable = true, configuration)
+      Success(configuration)
+    }
+    } recover { case e =>
+      if(configuration.usingInstanceRegistry) InstanceRegistry.sendMatchingResult(isElasticSearchReachable = false, configuration)
```

Comment:
Having the condition at all places where it is used can be dangerous. 
Could we always make the call and decide this in the method? (or through inheritance)